### PR TITLE
feature/remove-httpd-collector

### PR DESCRIPTION
**Pull request by jrouzierinverse**
# Description
Remove httpd collector

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* Remove unused service httpd.collector

### CODE REVIEW

**Review by satkunas — APPROVED** [review]
More UI changes pushed

File: debian/packetfence.conffiles [hunk 1]
```diff
@@ -1,75 +1,3 @@
-/etc/sudoers.d/packetfence
```
Comment (fdurand) [inline debian/packetfence.conffiles]: Why so many files ?

File: debian/patches/debianize.patch [hunk 2]
```diff
@@ -66,23 +66,6 @@ index 8045a372d3..7437d09bae 100644
...
-+++ b/conf/systemd/packetfence-httpd.collector.service
-@@ -10,9 +10,9 @@ After=packetfence-base.target packetfence-config.service packetfence-iptables.se
- StartLimitBurst=3
- StartLimitInterval=10
- Type=notify
--ExecStartPre=/usr/bin/rm -f /usr/local/pf/var/run/httpd.collector.pid
--ExecStartPre=/bin/perl -I/usr/local/pf/lib -I/usr/local/pf/lib_perl/lib/perl5 '-Mpf::services::manager::httpd_collector' -e 'pf::services::manager::httpd_collector->new()->generateConfig()'
--ExecStart=/usr/sbin/httpd -f /usr/local/pf/var/conf/httpd.conf.d/httpd.collector -DFOREGROUND  -Drhel
-+ExecStartPre=/bin/rm -f /usr/local/pf/var/run/httpd.collector.pid
-+ExecStartPre=/usr/bin/perl -I/usr/local/pf/lib -I/usr/local/pf/lib_perl/lib/perl5 '-Mpf::services::manager::httpd_collector' -e 'pf::services::manager::httpd_collector->new()->generateConfig()'
```
Comment (fdurand) [inline debian/patches/debianize.patch:81]: Since it's removed i don't understand why we have to patch the file

**Review by fdurand — CHANGES_REQUESTED** [review]

**Comment by jrouzierinverse** [comment]
done

**Comment by fdurand** [comment]
Rebase please

**Pull request merged**